### PR TITLE
[README] Programmatic usage + browserify

### PR DESCRIPTION
**Pull request by hiddentao**
Added information on programmatic usage and browserify bundling - took me a bit of time to figure this out and I hope it can help somebody else.

### CODE REVIEW

**Comment by imsky** [comment]
It needs to be cleaned up a bit (there are extra comments and whitespace), and the wording could be a bit more specific, but this seems good.

Why is there a section on a Browserify shim? Holder should work with Browserify out of the box.

**Comment by hiddentao** [comment]
I've removed the browserify section (I originally couldn't get it working unless I used the shim, but I just checked again now and it seems ok, so it must have been another issue).

Please let me know what other cleanup you'd like to see.

File: README.md [hunk 1]
```diff
@@ -52,6 +66,8 @@ Placeholder options are set through URL properties, e.g. `holder.js/300x200?x=y&
 * `align`: Custom text alignment (left, right). Example: `holder.js/300x200?align=left`
 * `outline`: Draw outline and diagonals for placeholder. Example: `holder.js/300x200?outline=yes`
 
+
+
```
Comment (imsky) [inline README.md]: no need for extra space

File: README.md [hunk 2]
```diff
@@ -38,6 +38,20 @@ The above tag will render as a placeholder 300 pixels wide and 200 pixels tall.
 
 To avoid console 404 errors, you can use ``data-src`` instead of ``src``.
 
+### Programmatic usage
+
+By default Holder will automatically process tags on the page. To programmatically insert a placeholder use the `run()` API:
```
Comment (imsky) [inline README.md]: don't need the first sentence

File: README.md [hunk 3]
```diff
@@ -38,6 +38,20 @@ The above tag will render as a placeholder 300 pixels wide and 200 pixels tall.
...
+
+```js
+// e.g. <img id="myImage" data-src="holder.js/300x200?auto=true" />
```
Comment (imsky) [inline README.md]: not necessary

File: README.md [hunk 4]
```diff
@@ -38,6 +38,20 @@ The above tag will render as a placeholder 300 pixels wide and 200 pixels tall.
...
+var myImage = document.getElementById('myImage');
+
+Holder.run({
+  images: myImage,
+  /* you can pass in additional options here like you normally would */
```
Comment (imsky) [inline README.md]: not necessary

**Comment by imsky** [comment]
@hiddentao comments added

**Comment by hiddentao** [comment]
I've made the changes.

**Comment by imsky** [comment]
Excellent, thanks very much.

**Pull request merged**